src/patterns/task-list-pages/index.md.njk

```diff
@@ -149,13 +156,6 @@ Register as a childminder
 
 ### Next steps
 
-Work is needed to ensure that users can understand the difference between completing a task and completing the whole service.
-
-In addition, more user research is needed to find out:
-
-- how to help screen reader users to get an overview of the progress they have made through the task list
-- whether to return users to the task list after each task or take them straight to the next task in the sequence
-- the best way to show when tasks must be completed in a fixed order
-- how to ensure users can see which tasks have been completed and which they still need to do
+A cross-government team are collaborating on work to update this pattern, and to introduce a component.
```

Comment:
@ImranH-GDS @frankieroberto 
How's this for a description of what's happening on the project and giving a call-to-action?

```suggestion
Since September 2021, a cross-government group have been collaborating to co-design an update to this pattern and introduce it as a component.

We're now working on [building the prototype component](https://github.com/alphagov/govuk-frontend/pull/2261).

This work is open to anyone wanting to help us. [Join the 'task-list-collab' Slack to find out how you can help](https://join.slack.com/t/task-list-collab/shared_invite/zt-us1bwvm8-VVemg6XFZFhdbCtedNXBSQ).
```

Reply:
@calvin-lau-sig7 sounds great to me.

Reply:
Thanks @frankieroberto! Would you mind leaving this 2 comments open (and not commiting) for a couple days whilst I let the team take a look at this?